cmd/minishift/cmd/addon/addon.go

```diff
@@ -23,15 +23,15 @@ import (
 const (
 	addOnConfigKey = "addons"
 
-	emptyAddOnError = "You must specify an add-on name. Use `minishift addons list` to view installed add-ons."
+	emptyAddOnError = "You must specify an add-on name. Run `minishift addons list` to view installed add-ons."
 	noAddOnMessage  = "No add-on with the name %s is installed."
 )
 
 var AddonsCmd = &cobra.Command{
 	Use:     "addons SUBCOMMAND [flags]",
 	Aliases: []string{"addon"},
-	Short:   "Manages Minishift add-ons",
-	Long:    "Manages Minishift add-ons. You can install, list, enable or disable Minishift add-ons.",
+	Short:   "Manages Minishift add-ons.",
+	Long:    "Manages Minishift add-ons. Use the sub-commands to install, list, enable or disable Minishift add-ons.",
```

Comment:
I think there is enough context here to indicate that the sub-commands belong to the parent command, especially if you see it in the --help. The word "its" is not recommended in docs because it's a possessive contraction and usually used to indicate ownership (and commands, unlike humans, can't "own" things). I already removed some instances of that word across the docs in past reviews.